Scripts/DCS-BIOS/lib/modules/aircraft_modules/C-130J.lua

```diff
@@ -222,6 +222,15 @@ local devices = {
 -- Center Console
 
 -- Pilot Lighting Panel
+local PLT_LIGHTING_PANEL = "Pilot Lighting Panel"
```

Comment:
The pattern we've been following since the tomcat (for better or for worse) has been to abbreviate the role name in the category, so `PLT`, `CPLT`, `WSO`, etc
```suggestion
local PLT_LIGHTING_PANEL = "PLT Lighting Panel"

```

Here's an example from the F-4E 

<img width="246" height="240" alt="Image" src="https://github.com/user-attachments/assets/7c587d2a-7017-4fa9-af1b-5596115747e3" />